javascript-data-model/order-history.js

```diff
@@ -0,0 +1,96 @@
+
+var orderHistory = ([
+  {
+    orderPlaced: 'Auguest 4, 2020',
+    total: 34.00,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-87722232',
+    deliveredDate: 'Delievered Aug,8 2020',
+    deliveryMethod: ' ',
+    item: {
+      description1: 'Javascript for Impatient programmers',
+      description2: '',
+      author: 'Rauschmayer Dr Axel',
+      rtrnWdindow: 'Sep 7,2020',
+      price1: 31.00,
+      price2: '',
+      buyItAgain: true,
+      viewItem: true,
+      productReview: true,
+      orderRetails: true,
+      invocie: true,
+      productSupport: false
+    }
+  },
+
+  {
+    orderPlaced: 'July 19,2020',
+    total: 44.53,
+    shipTo: 'JS Masher',
+    orderNum: '133-9984268-1280257',
+    deliveredDate: 'Delievered Jul 20, 2020',
+    deliveryMethod: 'Your package was delivered It was handed directly handed to a resident',
+    item: {
+      description1: 'The Timeless Way of Building',
+      description2: '',
+      author: 'Alexander Christopher',
+      rtrnWindow: 'Aug 19,2020',
+      price1: 41.33,
+      price2: '',
+      buyItAgain: true,
+      viewItem: true,
+      productReview: true,
+      orderDetails: true,
+      invocie: true,
+      productSupport: false
+    }
+  },
+
+  {
+    orderPlaced: 'July 4,2020',
+    total: 17.22,
+    shipTo: 'JS Masher',
+    orderNum: '114-2875557-9059409',
+    deliveredDate: 'Delievered Jul 7, 2020',
+    deliveryMethod: 'Your package was delivered.It was handed directly to a resident',
+    item: {
+      description1: 'Gamecube Contorller Adapter.Super Smash Bros Switch Gamecube Adapter fow Wii U Pc.Support Turbo and Vibration features.No Driver and No Lag-Gamecube Adapter',
+      description2: '',
+      author: '',
+      rtrnWindow: 'Aug 5,2020',
+      price1: 15.98,
+      price2: '',
+      buyItAgain: true,
+      viewYourItem: true,
+      productReview: true,
+      orderDetails: true,
+      invocie: true,
+      productSupprt: false
+    }
+  },
+
+  {
+    orderPlaced: 'July 3,2020',
+    total: 138.93,
+    shipTo: 'JS Masher',
+    orderNum: '113-2883177-2648248',
+    deliveredDate: 'Delievered Jul 5, 2020',
+    deliveryMethod: ' ',
+    item: {
+      description1: 'Gamecube Contorller-Super Smash Bros.edition (Nintendo Switch) Pc Support Turbo and Vibration features No Driver and No Lag -Gamecube Adapter',
+      description2: 'the Art of Sql',
```

Comment:
If you find yourself numbering properties like this, that generally means you should actually use an array. You should make the `item` property hold an array of objects, one for each item in the order.